EFAuditer/EFAuditServiceExtensions.cs

```diff
@@ -62,18 +64,21 @@ public static void AuditAction(AuditEvent ev, EventEntry entry, AuditLog audit)
                 ?? GetCustomKey(ev, CustomFields.AppUser)
                 ?? ev.Environment.UserName;
 
-            var serializerSettings = Audit.Core.Configuration.JsonSettings;
-            serializerSettings.Converters.Add(new Newtonsoft.Json.Converters.StringEnumConverter());
+            var serializerSettings = new JsonSerializerOptions
+            {
+                MaxDepth = 0,
+                IgnoreNullValues = true
+            };
 
             switch (audit.EventType)
             {
                 case "Insert":
                     audit.AuditData =
-                        JsonConvert.SerializeObject(entry.ColumnValues, serializerSettings);
+                        System.Text.Json.JsonSerializer.Serialize(entry.ColumnValues, serializerSettings);

```

Comment:
Would it be possible to refer to these bits as `JsonSerializer.Serialize` instead? This seems a bit long

Reply:
Yes, we were still importing Newtonsoft into this file so it was to differentiate when we were testing. Fixed